test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/006_ATF_P_TC_comm_app_first_StartService_RPC_encrypt_true.lua

```diff
@@ -10,12 +10,11 @@
 -- (meaning: SDL will NOT respond to StartService_request from mobile app till PTU will be finished per comment)
 --
 -- 1. Used preconditions:
--- RPC SetAudioStreamingIndicator is allowed by policy
 -- Communication app exists in LP, no certificate in module_config
 --
 -- 2. Performed steps
 -- 2.1. Register and activate application.
--- 2.2. Send StartService(serviceType = 7 (RPC), RPCfunctionID = 48(SetAudioStreamingIndicator))
+-- 2.2. Send StartService(serviceType = 7 (RPC))
```

Comment:
@istoimenova, Please remove "SDL should not process request to HMI" statement in line 23